arwen/elrondapi/bigintOps.go

```diff
@@ -558,9 +558,10 @@ func bigIntTDiv(context unsafe.Pointer, destination, op1, op2 int32) {
 	dest, a, b := bigInt.GetThree(destination, op1, op2)
 	if b.Sign() == 0 {
 		runtime := arwen.GetRuntimeContext(context)
-		runtime.SignalUserError("[bigIntTDiv] division by 0")
+		runtime.SignalUserError(arwen.UserErrorDivZero)
+	} else {
```

Comment:
no need for else. call return after signalusererror

Reply:
This came up from tests. Without the else the division gets executed and you get a panic. SignalUserError kills the vm but not the current function execution.